source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRInputToggleRenderer.mm

```diff
@@ -37,38 +38,47 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
 {
     std::shared_ptr<HostConfig> config = [acoConfig getHostConfig];
     std::shared_ptr<BaseCardElement> elem = [acoElem element];
-    std::shared_ptr<ToggleInput> toggleBlck = std::dynamic_pointer_cast<ToggleInput>(elem);
+    std::shared_ptr<ToggleInput> adaptiveToggleInput = std::dynamic_pointer_cast<ToggleInput>(elem);
 
     NSBundle *bundle = [NSBundle bundleWithIdentifier:@"MSFT.AdaptiveCards"];
     if (!bundle) { // https://github.com/Microsoft/AdaptiveCards/issues/1834
```

Comment:
>Microsoft [](start = 41, length = 9)

Nit: This looks to be a TODO; In that case, can you update teh comment to be more explicit following a "TODO: [Link to the tracking issue]  Log warnings and errors"